Index: 作业2.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># 作业二\r\n\r\n### 图1\r\n![第2次作业-图1](image/第2次作业-图1.png)\r\n\r\n### 图2\r\n![第2次作业-图2](image/第2次作业-图2.png)\r\n\r\n### 图3\r\n![第2次作业-图3](image/第2次作业-图3.png)\r\n\r\n### 图4\r\n![第2次作业-图4](image/第2次作业-图4.png)\r\n\r\n### 图5\r\n![第2次作业-图5](image/第2次作业-图5.png)\r\n\r\n### 图6\r\n![第2次作业-图6](image/第2次作业-图6.png)\r\n\r\n### 图7\r\n![第2次作业-图7](image/第2次作业-图7.png)\r\n
===================================================================
diff --git a/作业2.md b/作业2.md
--- a/作业2.md	(revision d72545211aee8ada1e8d97c4da7fb0a74ef5a6b1)
+++ b/作业2.md	(date 1775748601137)
@@ -1,22 +1,22 @@
 # 作业二
 
-### 图1
+# 图1
 ![第2次作业-图1](image/第2次作业-图1.png)
 
-### 图2
+# 图2
 ![第2次作业-图2](image/第2次作业-图2.png)
 
-### 图3
+# 图3
 ![第2次作业-图3](image/第2次作业-图3.png)
 
-### 图4
+# 图4
 ![第2次作业-图4](image/第2次作业-图4.png)
 
-### 图5
+# 图5
 ![第2次作业-图5](image/第2次作业-图5.png)
 
-### 图6
+# 图6
 ![第2次作业-图6](image/第2次作业-图6.png)
 
-### 图7
+# 图7
 ![第2次作业-图7](image/第2次作业-图7.png)
